pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -1859,30 +1868,44 @@ ParticipantStatusTrackingInfo.Kit:
       type: string
     shipper:
       type: string
-### StudyPasswordRequirements
-StudyPasswordRequirements:
+
+StudyPasswordPolicy:
   type: object
   required:
     - minLength
-    - isUppercaseLetterRequired
-    - isLowercaseLetterRequired
-    - isSpecialCharacterRequired
-    - isNumberRequired
+    - minCharClasses
```

Comment:
Is there a reason we need to abbreviate  `character` here? I don't think cutting the extra 5 characters saves us all that much, and it reads nicer . Same goes for `minimum`.

Reply:
Sure, we can use full word.